server/mqtt.go

```diff
@@ -1327,7 +1356,12 @@ func (s *Server) mqttCreateAccountSessionManager(acc *Account, quitCh chan struc
 				return nil, err
 			}
 		}
+		needToTransfer = false
+
+	default:
+		needToTransfer = si.Config.MaxMsgsPer != 1
```

Comment:
I would do > 1 here.

Reply:
`< 1` or `> 0` you mean?

Reply:
Actually it is correct since we control it, will either be 1 or default which could be 0 or -1 but in this case will be 0 if we have not transferred.